src/gui/guiItemImage.cpp

```diff
@@ -0,0 +1,43 @@
+
+#include "guiItemImage.h"
+
+GUIItemImage::GUIItemImage(gui::IGUIEnvironment *env, gui::IGUIElement *parent, s32 id,
+		const core::rect<s32> &rectangle, std::string item_name,
+		gui::IGUIFont *font, Client *client) :
+		gui::IGUIElement(gui::EGUIET_ELEMENT, env, parent, id, rectangle),
+		m_item_name(item_name), m_font(font), m_client(client),
+		m_label(core::stringw())
+{
+}
+
+void GUIItemImage::draw()
+{
+	if (m_client == 0) {
```

Comment:
`if (!m_client)` comparing a pointer with `0` works but is unusual.

Reply:
Done.
(This was copied from the old code.)